docs/F-Refinements/two-dimensional-arrays.md

```diff
@@ -7,22 +7,22 @@ sidebar_position: 5
 
 After reading this section, you will be able to:
 
-* Connect procedures using pass-by-value and pass-by-address semantics to build a complete program
+* Connect procedures using `pass-by-value` and `pass-by-address` semantics to build a complete program
 * Design data collections using arrays and structures to manage information efficiently
 
 ## Introduction
 
-A simple data structure for organizing tabular data is the two-dimensional array.  The C language supports multi-dimensional arrays.  The C compiler treats a two-dimensional array as an array of arrays.  An obvious application of this data structure is an array of character strings. 
+A simple data structure for organizing tabular data is the two-dimensional array. The C language supports multi-dimensional arrays. The C compiler treats a two-dimensional array as an array of arrays. An obvious application of this data structure is an array of character strings. 
 
-This chapter introduces two-dimensional arrays, describing their syntax and their organization in memory.  This chapter includes sample code snippets related to arrays of character strings. 
+This chapter introduces two-dimensional arrays, describing their `syntax` and their `organization` in memory. This chapter includes sample code snippets related to arrays of character strings. 
 
 ## Two-Dimensional Syntax
 
-A table is a useful analogy for describing a two-dimensional array.  An entry in a table is identified by its row and column positions.  Consider the row and column indices in the figure below.  The first index refers to the row and the second index refers to the column.
+A table is a useful analogy for describing a two-dimensional array. An entry in a table is identified by its row and column positions. Consider the row and column indices in the figure below. `The first index` refers to the `row` and `the second index` refers to the `column`.
 
-![](https://ict.senecacollege.ca//~ipc144/pages/images/2d%20array.png)
+![2dArrayExample](https://ict.senecacollege.ca//~ipc144/pages/images/2d%20array.png)
```

Comment:
In discussion with project maintainers, I think they agree on having more descriptive alt tags for the images.
Ex: [a 5x5 table, describing two-dimensional arrays